powerquery-docs/HelperFunctions.md

```diff
@@ -171,7 +171,10 @@ Table.GenerateByPage = (getNextPage as function) as table =>
         // otherwise set the table type based on the columns of the first page
         if (firstRow = null) then
             Table.FromRows({})
-        else        
+		// check for empty first table
```

Comment:
These lines should be unindented one level. (This applies to the existing TripPin versions of this code as well.)

Reply:
Good catch. Done!